.travis.yml

```diff
@@ -1,12 +1,16 @@
-sudo: false
+sudo: required
 
+language: node_js
+node_js:
+  - 10
 os:
   - linux
 
 before_install:
   - if [ $TRAVIS_OS_NAME == "linux" ]; then
-      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
-      sh -e /etc/init.d/xvfb start;
+      export CXX="g++-4.9" CC="gcc-4.9";
+      export DISPLAY=':99.0'
+      /usr/bin/Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
```

Comment:
I tried mirroring vscode-test's travis config, but this didn't seem to do anything different